catalog/core/catalog-core-commands/pom.xml

```diff
@@ -199,22 +199,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.39</minimum>
+                                            <minimum>0.30</minimum>
```

Comment:
These coverage numbers were pretty low to begin with; if possible, try to add some unit tests.